src/client/java/teammates/client/scripts/sql/VerifyCourseEntityAttributes.java

```diff
@@ -154,14 +163,180 @@ private boolean verifyStudent(CourseStudent oldStudent,
     }
 
     private List<Student> getNewStudents(String courseId) {
-        HibernateUtil.beginTransaction();
+        // HibernateUtil.beginTransaction();
         CriteriaBuilder cb = HibernateUtil.getCriteriaBuilder();
         CriteriaQuery<teammates.storage.sqlentity.Student> cr = cb
                 .createQuery(teammates.storage.sqlentity.Student.class);
         Root<teammates.storage.sqlentity.Student> courseRoot = cr.from(teammates.storage.sqlentity.Student.class);
         cr.select(courseRoot).where(cb.equal(courseRoot.get("courseId"), courseId));
         List<Student> newStudents = HibernateUtil.createQuery(cr).getResultList();
-        HibernateUtil.commitTransaction();
+        // HibernateUtil.commitTransaction();
         return newStudents;
     }
+
+    // methods for verify feedback chain -----------------------------------------------------------------------------------
+
+    private boolean verifyFeedbackChain(teammates.storage.sqlentity.Course newCourse) {
+        List<teammates.storage.sqlentity.FeedbackSession> newSessions = newCourse.getFeedbackSessions();
+        List<FeedbackSession> oldSessions = ofy().load().type(FeedbackSession.class)
+                .filter("courseId", newCourse.getId()).list();
+
+        if (newSessions.size() != oldSessions.size()) {
```

Comment:
We can leave this here but counts should have been checked in `VerifyCourseEntityCounts`